test/c/test_substrait_c_api.cpp

```diff
@@ -45,3 +46,23 @@ TEST_CASE("Test C Get and To Json-Substrait API", "[substrait-api]") {
 
   REQUIRE_THROWS(con.FromSubstraitJSON("this is not valid"));
 }
+
+
+TEST_CASE("Test C VirtualTable input Literal", "[substrait-api]") {
+  DuckDB db(nullptr);
+  Connection con(db);
+
+  auto json = con.GetSubstraitJSON("select * from (values (1, 2),(3, 4))");
+  REQUIRE(!json.empty());
+  std::cout << json << std::endl;
```

Comment:
Do we want to test anything else besides that there wasn't an error during parsing?  Perhaps call GetSubstrait and check a few things like the number of rows/columns is right, that the type of the read operation is a virtual table, etc.

Reply:
It will done as part of from_substrait PR. Waiting on a change from duckdb (Constructor having RelationContextWrapper and ParsedExpression) once it is in, same test case will be enhanced to check roundtrip